StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -134,44 +107,27 @@ class AddPaymentMethodViewController: UIViewController {
 
     private let intent: Intent
     private let configuration: PaymentSheet.Configuration
+    private let isLinkEnabled: Bool
```

Comment:
Maybe naive: But can we read is link enabled from the intent?

Reply:
Great q, a lot of code assumed this incorrectly and I just fixed it! We can't; whether link is enabled on the Intent is not the same as whether it's enabled for us. e.g. we disable Link if the merchant requires billing address.

Reply:
Makes sense, thanks!